service/resource/configmap/update_test.go

```diff
@@ -426,3 +432,81 @@ func Test_Resource_ConfigMap_ProcessUpdateState(t *testing.T) {
 		}
 	}
 }
+
+// Test_Resource_ConfigMap_Reload tests that the configmap is reloaded correctly.
+func Test_Resource_ConfigMap_Reload(t *testing.T) {
+	configMapKey := "prometheus.yml"
+	configMapName := "prometheus"
+	configMapNamespace := "monitoring"
+
+	handlerReceivedMessage := false
+	testServer := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		method := r.Method
+		if method != "POST" {
+			t.Fatalf("incorrect method used: %s\n", method)
+		}
+
+		path := r.URL.Path
+		if path != "/-/reload" {
+			t.Fatalf("incorrect path used: %s\n", path)
+		}
+
+		handlerReceivedMessage = true
```

Comment:
If there is a flag anyway, I think memorising request and inspecting it later is nicer.

Reply:
Fixed